docs/src/api/javascript.md

```diff
@@ -0,0 +1,16 @@
+### param: ElementHandle.$eval.pageFunction = %%-js-evalonselector-pagefunction-%%
```

Comment:
We should not allow overriding by alias, this is confusing. Let's always use original name instead.

Reply:
Done.